src/Humanizer/TimeSpanHumanizeExtensions.cs

```diff
@@ -8,190 +8,235 @@
 
 namespace Humanizer
 {
+  /// <summary>
```

Comment:
Doesn't seem to obey the [`.editorconfig`](https://github.com/Humanizr/Humanizer/blob/766a10b/.editorconfig), stating 4 spaces.

This makes the diff much larger than it's supposed to.